CalcualtorLogic.js

```diff
@@ -138,23 +138,13 @@
             this.resultElement.textContent = this.currentResult;
         }
         onOperatorButtonClick(event) {
-            if (this.lastValue && this.currentResult && this.operator !== null) {
-                let firstOperand = parseFloat(this.lastValue);
-                let secondOperand = parseFloat(this.currentResult);
-                let _result;
-                if (this.operator === "+") _result = this.calculatorLogic.add(firstOperand, secondOperand);
-                if (this.operator === "-") _result = this.calculatorLogic.subtract(firstOperand, secondOperand);
-                if (this.operator === "*") _result = this.calculatorLogic.multiply(firstOperand, secondOperand);
-                if (this.operator === "/") _result = this.calculatorLogic.divide(firstOperand, secondOperand);
-                this.currentResult = _result;
-            }
+            this.evaluate();
```

Comment:
You could call `this.onEqualButtonClick()` also.
but what you did is fine, don't change it.